### DISCUSSION TITLE
i cant do bookmarklet pls help

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#939] [(B) Bug Discussion]
i cant do bookmarklet hacks let say i paste _.player.modifiers.maxHearts = Infinity; in the thing the click genarate drag it to the top and log on to prodigy i click on it and nothing happens i frag it on to my screen and it opens up new tad that say error#blocked do i need redirector if i do what version???

**Comment by ghost** [comment]
can you pls help @PatheticMustan

**Comment by ghost** [comment]
yea same allso id like to report https://gamehack.icu/prodigygame im pretty shure they stole r hacks

**Reply by PatheticMustan** [reply]
thanks, that's not even stealing our hacks, it's just a survey scam or something

**Reply by ghost** [reply]
ooooh but somone i know actully got a hacked acount with epics and trialsmasters and level 1e+67 and all pets and items and buddies

**Reply by ghost** [reply]
good

**Comment by LeoBadeaux** [comment]
No. Bookmarklet will not work at all unless you have the extension. Please don't mention specific staff members, somebody will get to you. You opened a thread  very similar to this recently, in which I replied no to, do not continue this behaviour.
